app/styleguides/_default/components/subheader.css

```
@@ -5,4 +5,43 @@ $helvetica-stack: Helvetica, sans-serif;
   color: $black;
   font: bold 22px $helvetica-stack;
   margin: 0 0 1em;
+  .h4 {
+    font-size: 22px;
+    line-height: 1.09;
+  }
+
+  .h3 {
+    font-size: 28px;
+    line-height: 1.09;
+    & > .link {
+      padding: 3px 0px 0px 5px;
```

We can omit the unit when using `0` as value
```suggestion
      padding: 3px 0 0 5px;
```